website/docs/getting-started/setup/instance-management/update-appsmith.md

```diff
@@ -0,0 +1,53 @@
+---
+description: Follow the steps to update Appsmith to the latest version.
+---
+
+# Update Appsmith
+This page provides instructions to update Appsmith to the latest version.
+
+## Prerequisites
+Ensure that you complete the following steps before starting the update process.
+
+* For backup and update tasks, you need at least 2 GB of free storage.
+* You need to create a backup of the Appsmith instance before a manual update. To create a backup, see the [Backup Appsmith Instance](/getting-started/setup/instance-management/appsmithctl#backup-appsmith-instance) section.
+* If you are on a version earlier than v1.9.2, first upgrade to version v1.9.2. To upgrade, see the [Checkpoint version and upgrade](/getting-started/setup/instance-management#checkpoint-version-and-upgrades) section.
+
+## Docker Compose
+Follow the below steps on Docker or platforms that use `docker-compose`. For example, AWS AMI or DigitalOcean.  
+
+You can update Appsmith in one of the following ways:
+
+* [Manual update](#manual-update)
+* [Automatic update](#automatic-update)
+
+### Manual update
+Go to the root directory of your installation and run:
+
```

Comment:
```suggestion
:::info
If you do not remember the directory of you Appsmith installation, run:
`docker inspect -f '{{ (index .Mounts 0).Source }}’ <your-appsmith-container-id>`
::::
```

Reply:
The above command only gives the user the installation directory location on their machine, but this uses `docker` and not `docker-compose`.  Do we have a docker-compose equivalent as we are only using docker-compose commands in the documentation?

Reply:
> The above command only gives the user the installation directory location on their machine, but this uses `docker` and not `docker-compose`. Do we have a docker-compose equivalent, as we are only using docker-compose commands in the documentation? Also, this command is not returning the location for me. I'm getting this error:
> 
> ```
> "docker inspect" requires at least 1 argument.
> See 'docker inspect --help'.
> 
> Usage:  docker inspect [OPTIONS] NAME|ID [NAME|ID...]
> ```

Fixed it. The command had a backtick after the format string.  

`docker inspect -f '{{ (index .Mounts 0).Source }}' <your-appsmith-container-id>`

Reply:
This command is applicable to docker-comose installations as well.